tests/unit/profiles-management/helpers/test_kfam.py

```diff
@@ -34,3 +51,62 @@ def test_non_profile_owner_resource():
     resource = GenericNamespacedResource(metadata=ObjectMeta(name="random"))
 
     assert not resource_is_for_profile_owner(resource)
+
+
+def test_contributor_getters():
+    user = "test"
+    role = ContributorRole.ADMIN
+    resource = GenericNamespacedResource(
+        metadata=ObjectMeta(name="test", annotations={"role": role, "user": user})
+    )
+
+    assert get_contributor_user(resource) == user
+    assert get_contributor_role(resource) == role
+
+
+def test_rolebinding_not_matching_empty_profile_contributors():
```

Comment:
Similar to my other comments, this test case can be parametrised.

Reply:
Yep!
4ec31d0